bigtable-hbase-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/filters/BigtableWhileMatchResultScannerAdapter.java

```diff
@@ -33,6 +38,7 @@
  * @author sduskis
  * @version $Id: $Id
  */
+@SuppressWarnings("deprecation")
```

Comment:
can you add a comment why we are suppressing deprecation as opposed to removing it?

Reply:
will do.  At a high level, NotThrowingClosable is compatible with old versions of opencensus.  Newer versions use something else. I wanted to make sure that we have the least chance of problems when working with other GCP libraries that may use older versions.